lib/rules/selector-max-specificity/index.mjs

```diff
@@ -237,7 +108,16 @@ const rule = (primary, secondaryOptions) => {
 			flattenNestedSelectorsForRule(ruleNode, result).forEach(({ selector, resolvedSelectors }) => {
 				resolvedSelectors.forEach((resolvedSelector) => {
 					// Check if the selector specificity exceeds the allowed maximum
-					if (compare(nodeSpecificity(resolvedSelector), maxSpecificity) > 0) {
+					if (
+						compare(
+							selectorSpecificity(resolvedSelector, {
+								customSpecificity: secondaryOptions?.ignoreSelectors
+									? isSelectorNodeIgnored
+									: undefined,
+							}),
+							maxSpecificity,
+						) > 0
+					) {
```

Comment:
suggestion
refactor

prettier is getting funky with the formatting
maybe set `const customSpecificity = secondaryOptions?.ignoreSelectors ? isSelectorNodeIgnored : undefined;` before the `if`?

Reply:
I've made a few more changes in the same spirit.
Let me know what you think